.github/workflows/build-ios-release-pullrequest.yml

```
@@ -86,50 +93,64 @@ jobs:
             ${{ runner.os }}-provisioning-profiles-
+
       - name: Check Cache Status for Provisioning Profiles
         run: |
           if [ -n "${{ steps.cache_provisioning_profiles.outputs.cache-hit }}" ]; then
             echo "Provisioning profiles were restored from cache."
           else
             echo "No cache found for provisioning profiles. A new cache will be created."
           fi
+
       - name: Verify Provisioning Profiles Exist
         run: |
```

actionlint

shellcheck reported issue in this script: SC2088:warning:1:10: Tilde does not expand in quotes. Use $HOME

(shellcheck)